src/guide/1/getting-started/first-step/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: full.html.hbs
+algolia: true
+title: Kuzzle
+description: Getting started with Kuzzle
+---
+
+# Getting Started with Kuzzle
+
+In this tutorial you will learn how to install and run **Kuzzle**.
+
+## Running Kuzzle
+
+In this section we'll learn how to quickly get Kuzzle up and running using our installation script.
```

Comment:
Inconsistency with the tone use in the previous paragraph above. Either you use "you" in this page, or you use "we", but not both.

This comment is to be applied to this entire document.